config/routes.rb

```diff
@@ -67,6 +67,7 @@
 
   # Legacy URLs
   get "/prisoner-details", to: redirect("/prisoner")
+  get "/deferred/confirmation/new" => "confirmations#new"
```

Comment:
Someone processing booking requests follows a link from an email and ends up on the URL they expected from the link.

Whilst Rails's routing does support redirection, it discards the query-string parameters, so we'd have to do something in a controller. I'm not sure it's worth that additional complexity, and I'm not convinced it's any improvement from a user's point of view.